src/pages/tutorials/smart-rollups/run.md

```diff
@@ -0,0 +1,108 @@
+---
+id: run
+title: "Part 6: Running the rollup node"
+lastUpdated: 11th October 2023
+---
+
+Now that the smart rollup is originated on layer 1, anyone can run a smart rollup node for it.
+Smart rollup nodes are similar to baking nodes, but they run the smart rollup kernel instead of baking Tezos blocks.
+In these steps, you start a smart rollup node, but note that anyone can run a node based on your kernel, including people who want to verify the rollup's behavior.
+
+1. Copy the contents of the `preimages` folder to a folder that the rollup node can access by running these commands:
+
+   ```bash
+   mkdir -p ~/.tezos-rollup-node/wasm_2_0_0
+
+   cp preimages/* ~/.tezos-rollup-node/wasm_2_0_0/
+   ```
+
+1. In the second terminal window, in the Docker container, start the rollup node:
+
+   ```bash
+   octez-smart-rollup-node-alpha run operator for "test_smart_rollup" with operators "bootstrap2" --data-dir ~/.tezos-rollup-node/ --log-kernel-debug --log-kernel-debug-file hello_kernel.debug
+   ```
+
+   Now the node is running and writing to the log file `hello_kernel.debug`.
+   Leave this command running in the terminal window just like you left the first terminal window running the Tezos sandbox.
+
+1. Open a third terminal window and enter the Docker container again:
+
+   ```bash
+   docker exec -it octez-container /bin/sh
+   ```
+
+1. In the container, go to the `hello_world_kernel` folder.
+
+1. Print the contents of the log file:
+
+   ```bash
+   tail -f hello_kernel.debug
+   ```
+
+   Now, each time a block is baked, the smart rollup node prints the contents of the messages in the smart rollup inbox, as in this example:
+
+   ```
+   # Hello, kernel!
+   # Got message: Internal(StartOfLevel)
+   # Got message: Internal(InfoPerLevel(InfoPerLevel { predecessor_timestamp: 2023-06-07T15:31:09Z, predecessor: BlockHash("BLQucC2rFyNhoeW4tuh1zS1g6H6ukzs2DQDUYArWNALGr6g2Jdq") }))
+   # Got message: Internal(EndOfLevel)
+   ```
+
+1. Stop the command by pressing Ctrl + C.
+
+1. Run this command to watch for external messages to the rollup:
+
+   ```bash
+   tail -f hello_kernel.debug | grep External
+   ```
+
+   No output appears at first because the rollup has not received any messages aside from the internal messages that indicate the beginning and end of the inbox.
+
+   Leave this command running.
+
+1. Open a fourth terminal window, enter the Docker container with the command `docker exec -it octez-container /bin/sh`, and go to the `hello_world_kernel` folder.
```

Comment:
I would suggest to extract the part starting here in a separate page "Interacting with the node" or something, because this goes beyond just running the node, and constitutes the ultimate goal of the tutorial. Currently, it seems that the final step "Running the node" is the ultimate goal, but running the node is only useful if you actually do something with it, isn't it?

Reply:
Splitting this section would create two very short sections. I divided this page into two parts named "Running a smart rollup node" and "Interacting with the rollup node." How does that sound?

Reply:
That's fine too, thanks